example/CMakeLists.txt

```diff
@@ -7,8 +7,8 @@ cmake_minimum_required(VERSION 3.16)
 
 # Use the project template to get the settings required for an application core project
 set(${PROJECT_NAME}_MAJOR_VERSION 01)
-set(${PROJECT_NAME}_MINOR_VERSION 00)
-set(${PROJECT_NAME}_PATCH_VERSION 00)
+set(${PROJECT_NAME}_MINOR_VERSION 02)
```

Comment:
why version change 1.0.0 -> 1.2.3?

Reply:
I had this, so I could see what happened. Also, there is a lot that has been added recently. We should at least up the patch version but I thought it is enough to make it minor.

Reply:
yes, so please make it `1.1.0`. (I understand your need to change it to 1.2.3 for testing purposes, but that should probably not get committed, unless we do not actually want to do any version counting for the example and always keep the example at 1.2.3 for exactly this purpose. In that case I would suggest to take rather mad numbers like 12.34.56)

Reply:
To make sure the leading zero conversion is not broken, I would go for `01.23.45-67`

Reply:
sounds good. please add a comment that this version is fixed and does not represent the version of the example and must be changed for a real application (when using the example as a template).